src/App.css

```diff
@@ -0,0 +1,288 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+/* width */
+::-webkit-scrollbar {
+  width: 8px;
+  height: 20px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: var(--color-primary);
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: var(--color-primary-dark);
+}
+
+:root {
+  font-family: 'Lato', sans-serif;
+
+  --color-primary: #ec4c8a;
+  --color-primary-dark: #b21451;
+  --color-primary-light: #cf4278;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: var(--color-primary-dark);
+}
+
+body {
+  background-color: var(--color-primary);
+  margin: 0;
+}
+
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+/* *,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+} */
+
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
Well resolved 👍🏽

Reply:
- [ ] [OPTIONAL] Kindly remove the unused lines of code as recommended by the previous reviewer for a clean code base.